tools/regs2groov.py

```diff
@@ -0,0 +1,134 @@
+import xml.etree.ElementTree as et
+import re
+from collections import namedtuple
+import argparse
+
+
+Group = namedtuple('Group', ['name', 'registers'])
+Register = namedtuple('Register', ['name', 'address', 'fields'])
+Field = namedtuple('Field', ['name', 'access', 'msb', 'lsb'])
```

Comment:
**praise**: I like named tuples, I need to use more of them. (Instead of nameless)